Automattic/wp-calypso
# Store: Stripe Connect: Wire the disconnect link through redux

See #17002

To test:
* Follow the steps in #19115 to setup your environment
* **Edit: Important: Use the fix/1203-explicit-status-stripe-rest-api-errors branch when cloning woocommerce-services**
* After you've used the test bench to connect the site, navigate to http://calypso.localhost:3000/store/settings/{DOMAIN}
* Click on the Manage link for Stripe
* Wait for the placeholder dialog to be replaced by a dialog showing you're connected, and which includes a Disconnect link
* Click the Disconnect link
* Verify you are disconnected and dropped on the key based flow dialog
* Force refresh the page and enter Manage again
* Verify you are dropped on the key based flow

* Run `npm run test-client client/extensions/woocommerce/state/` and verify all tests pass
